### PR TITLE
Feature 1078/issue post request for new ad hoc template

### PR DESCRIPTION
**Pull request by pieperm**
Dependent on pull request #1080 and pull request #1103 

To test this, go to `/feedback/request` and create a new ad-hoc template. This should save the template to the database, display it on the page, and automatically select it in the URL.

Closes issue #1078

### CODE REVIEW

**Review by JEdwardBrown — COMMENTED** [review]
TeamSummaryCard.jsx, you removed the div at 150 and 173, and replaced with with an empty tag. Shouldn't it be a React.Fragment?

File: web-ui/src/components/feedback_template_selector/FeedbackTemplateSelector.jsx [hunk 1]
```diff
@@ -81,35 +135,36 @@ const FeedbackTemplateSelector = ({changeQuery}) => {
...
+          variant="contained"
+          color="primary"
+          onClick={onNewAdHocClick}>
+          New Ad-Hoc Template
+        </Button>
+        <Tooltip title="An ad-hoc template allows you to ask a single question" arrow>
+          <HelpOutlineIcon style={{color: "gray", marginLeft: "10px"}}/>
+        </Tooltip>
+      </div>
       <div className="card-container">
```
Comment (zsbrown97) [inline web-ui/src/components/feedback_template_selector/FeedbackTemplateSelector.jsx:154]: Whenever I load up the feedback page it throws a TypeError saying that templates is undefined in FeedbackTemplateSelector.jsx.

Comment (pieperm) [reply web-ui/src/components/feedback_template_selector/FeedbackTemplateSelector.jsx:193]: I could not figure out how to reproduce this error, please let me know if it's still happening after the most recent changes.

Comment (bhatiam12) [reply web-ui/src/components/feedback_template_selector/FeedbackTemplateSelector.jsx:210]: @zsbrown97 are you still seeing the error ?

**Review by moshirim — APPROVED** [review]

**Review by sagetheherb — APPROVED** [review]

**Review by zsbrown97 — APPROVED** [review]

**Pull request merged**